applications/MetisApplication/tests/quad_partition_test.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from KratosMultiphysics import KratosUnittest, Model, ModelPartIO, PARTITION_INDEX, FRACTIONAL_STEP
+from KratosMultiphysics.kratos_utilities import DeleteDirectoryIfExisting
+from KratosMultiphysics.MetisApplication import MetisDivideHeterogeneousInputProcess
+
+class QuadPartitionTest(KratosUnittest.TestCase):
+
+    def testConditionPartitioning(self):
+        test_file = "quads"
+        partitions = 3
+        sync_conditions = True
+        domain_size = 2
+        verbosity = 0
+
+        work_dir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I recommend using the pathlib module
e.g. sth like this

```suggestion
        work_dir = Path(__file__).parent.absolute
```